src/applications/pre-need/config/form.jsx

```diff
@@ -561,6 +561,13 @@ const formConfig = {
                 desiredCemetery: autosuggest.uiSchema(
                   'Which VA national cemetery would you prefer to be buried in?',
                   getCemeteries,
+                  {
+                    'ui:options': {
+                      widgetProps: {
```

Comment:
I would change the name of this to `inputProps`. If we were going for completeness, the maximal version of this might be something like (where you would only be interested in adding to `inputProps` for this PR):
```js
'ui:options': {
    inputProps: { ... },
    itemProps: { ...},
    labelProps: { ... },
    menuProps: { ... },
    rootProps: { ... },
},
```

There is documentation about the different prop getters [here](https://www.npmjs.com/package/downshift#prop-getters)